.gitattributes

```diff
@@ -0,0 +1,11 @@
+/tests export-ignore
+/docs export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/.scrutinizer.yml export-ignore
+/*.dist export-ignore
+/phpunit.xml export-ignore
+
+# Line endings
+sake text eol=lf
```

Comment:
this shouldn't be needed - `sake` doesn't exist in this repo...

Reply:
you're right, I'll remove this line for all but framework